torchgeo/transforms/transforms.py

```diff
@@ -272,3 +272,54 @@ def apply_transform(
             out = rearrange(out, 'b t c h w -> (b t) c h w')
 
         return out
+
+
+class _Clamp(K.IntensityAugmentationBase2D):
```

Comment:
@johnnv1 is this something Kornia would be interested in? If so I can contribute it when I have a chance.

Reply:
I think so, not sure why it will be a augmentation. But i think it's fine to add to kornia if have uses for it. Just to make sure, @shijianjian we don't have it, right?

Reply:
The use case is that this comes up commonly in satellite imagery pipelines where images aren't always uint8 and need to be converted to uint8 (0–255) or normalized (0–1) without out-of-bounds data. See [SeCo](https://github.com/ServiceNow/seasonal-contrast/blob/8285173ec205b64bc3e53b880344dd6c3f79fa7a/datasets/seco_dataset.py#L53) and [Satlas](https://github.com/allenai/satlas/blob/main/Normalization.md#sentinel-2-images) for examples where this is used.